src/mame/heathkit/tlb.cpp

```diff
@@ -1056,6 +1052,10 @@ void heath_gp19_tlb_device::device_start()
 	m_graphic_mode = false;
 	m_col_132 = false;
 	m_reverse_video = false;
+
+	// Allow proper handling of Interlace/video mode of CRTC
+	// This is needed to properly display 50 line mode.
+	m_crtc->set_allow_interlace_video_mode(true);
```

Comment:
You should be able to move this directly in the CRTC parameter setup on top, in `device_add_mconfig`